apps/db/lib/db/resource_history.ex

```diff
@@ -5,6 +5,7 @@ defmodule DB.ResourceHistory do
   use Ecto.Schema
   use TypedEctoSchema
 
+  @derive {Jason.Encoder, except: [:__meta__, :__struct__]}
```

Comment:
`only:` (opt-in) plutôt que `except:` (opt-out) nous aurait permis si je comprends bien de n'avoir que les champs qu'on veut dans la sortie, et ça me paraît préférable pour éviter de se mettre à publier des champs par défaut sans en avoir conscience (surtout que ça change le contrat externe).

Est-ce que ça impacte bien ça ?

Par ailleurs il aurait été bien d'avoir un test avant de mettre le fix, pour être sûr de ne pas régresser après, mais vu que tout est cassé, on pourrait le faire dans une PR suivante (ou là, comme tu le sens!).